Process.c

```diff
@@ -477,6 +477,8 @@ long Process_compare(const void* v1, const void* v2) {
       return (p1->processor - p2->processor);
    case SESSION:
       return (p1->session - p2->session);
+   case TREE_SET:
+      return (p1->tree_lft < p2->tree_lft) ? -1 : 1;
```

Comment:
Why not `p1->tree_lft - p2->tree_lft`?

Reply:
Actually it's not the same. 
It turned out it that it's `p2->tree_lft - p1->tree_lft`